components/Form/Select/Select.tsx

```diff
@@ -30,50 +32,55 @@ const Select = ({
   return (
     <div
       style={style}
-      className={cx(`govuk-form-group govuk-grid-column-${govGrid}`, {
-        'govuk-form-group--error': error,
-      })}
+      className={cx(
+        `govuk-form-group lbh-form-group govuk-grid-column-${govGrid}`,
+        {
+          'govuk-form-group--error': error,
+        }
+      )}
     >
-      <label className={`lbh-label govuk-label--${labelSize}`} htmlFor={name}>
+      <label className={`lbh-label`} htmlFor={name}>
```

Comment:
to get the spacing perfect, this may need one more class on the label. refer to: https://design-system.hackney.gov.uk/components/select/